docs/networks/testnet.md

```diff
@@ -0,0 +1,111 @@
+# Testnet Configuration
+
+The Graph Network's testnet is on Goerli, and network information can be found at https://testnet.thegraph.com/explorer/network.
+
+## Latest Releases
+
+| Component       | Release                                                                                    |
+| --------------- | ------------------------------------------------------------------------------------------ |
+| contracts       | [1.13.0](https://github.com/graphprotocol/contracts/releases/tag/v1.13.0)                  |
+| indexer-agent   | [0.20.5-alpha.1](https://github.com/graphprotocol/indexer/releases/tag/v0.20.5-alpha.1)    |
+| indexer-cli     | [0.20.5-alpha.1](https://github.com/graphprotocol/indexer/releases/tag/v0.20.5-alpha.1)    |
+| indexer-service | [0.20.5-alpha.1](https://github.com/graphprotocol/indexer/releases/tag/v0.20.5-alpha.1)    |
+| graph-node      | [0.28.2](https://github.com/graphprotocol/graph-node/releases/tag/v0.28.2)                 |
+
+## Approving And Staking
```

Comment:
Remove `Approving And Staking` section from this file and place it in the testnet-setup.md?`

Reply:
Makes sense!